src/site/content/en/blog/add-touch-to-your-site/index.md

```diff
@@ -0,0 +1,810 @@
+---
+title: Add touch to your site
+subhead: |
+  Touchscreens are available on more and more devices,  from phones up to desktop screens. Your app should respond to their touch  in intuitive and beautiful ways.
+date: 2014-01-01
+updated: 2018-09-20
+authors:
+  - mattgaunt
+description: |
+  Touchscreens are available on more and more devices,  from phones up to desktop screens. Your app should respond to their touch  in intuitive and beautiful ways.
+tags:
+  - blog
+---
+
+{% YouTube id="Rwc4fHUnGuU" %}
+
+
+Touchscreens are available on more and more devices, ranging from phones to
+desktop screens. When your users choose to interact with your UI, your app
+should respond to their touch in intuitive ways.
+
+## Respond to element states
+
+Have you ever touched or clicked an element on a web page and questioned
+whether the site actually detected it?
+
+Simply altering the color of an element as users touch or interact with parts
+of your UI gives a basic reassurance that your site is working. Not only does
+this alleviate frustration, it can also give a snappy and responsive feel.
+
+DOM elements can inherit any of the following states: default, focus, hover
+and active. To change our UI for each of these states, we need to apply styles
+to the following pseudo classes `:hover`, `:focus` and `:active` as shown below:
+
+```css
+.btn {
+  background-color: #4285f4;
+}
+
+.btn:hover {
+  background-color: #296CDB;
```

Comment:
```suggestion
  background-color: #296cdb;
```

Reply:
Noticed some case inconsistency with regard to letters in hex colors. Defaulting to lowercase I think is preferred in our code style guide.